src/components/item/BulkItemsForm.vue

```diff
@@ -77,17 +94,31 @@ export default {
         });
       };
 
-      const items = this.delimiter
+      let items = this.delimiter
         ? this.bulkItemsString.split(this.delimiter)
         : [this.bulkItemsString];
 
-      this._addBulkItemsOnServer({
-        items: items.map(item => makeItemObj(trimItemTitle(item))),
-        listId: this.currentListId,
-      })
-        .then(() => {
-          this.$vfm.hide('bulkItemsModal');
-        });
+      items = items.map(item => makeItemObj(trimItemTitle(item)));
+
+      if (items.length > this.$options.BULK_ITEMS_MAX_COUNT) {
+        this.titlesErrorMessage = this.titlesErrorMessage
+          ? this.titlesErrorMessage += `; also, ${this.$options.BULK_ITEMS_COUNT_ERROR_MESSAGE}`
```

Comment:
it's really  hard to notice that in teranary here you have changes.
better not to do that kind of operations in ternary

ahhhh
you have here double changing

you need here just +, not +=